dags/airflow_db_prune_logs_rotate.py

```diff
@@ -0,0 +1,140 @@
+import os
+import pendulum
+
+from airflow.decorators import dag, task
+from airflow.models import Param
+from airflow.utils.log.logging_mixin import LoggingMixin
+
+from utils.slack_operator import task_fail_slack_alert
+
+DEPLOYMENT_ENVIRONMENT = os.getenv("ENVIRONMENT")
+
+
+default_task_args = {
```

Comment:
We generally call these `default_args` and I've always thought that they were default arguments to the `@dag` decorator .. not true. They are default task arguments. I renamed this symbol to reflect that a-ha moment.